components/code-editor/index.js

```diff
@@ -21,7 +21,7 @@ function loadScript() {
 		}
 
 		const script = document.createElement( 'script' );
-		script.src = `/wp-admin/load-scripts.php?load=${ handles.join( ',' ) }`;
+		script.src = `${ wpApiSettings.schema.url }/wp-admin/load-scripts.php?load=${ handles.join( ',' ) }`;
```

Comment:
It's probably not the only place where we use this, but we should move away from using globals. I'm also wondering if the `CodeEditor` belongs in the `components` module since the components module is supposed to be for generic components (without WP dependency?).

Anyway, this is not urgent and will have to be figured out when we move components to an npm dependency anyway. And this is not specific to this component.